### DISCUSSION TITLE
Branding: is the app "draw.io" or "diagrams.net"?

### DISCUSSION THREAD

**Discussion by borekb** [jgraph/drawio#3579] [General]
I though that since around 2000, the app is called "Diagrams.net", not "Draw.io", for example, the Wikipedia page is https://en.wikipedia.org/wiki/Diagrams.net, not ~~https://en.wikipedia.org/wiki/Draw.io~~.

However, I can't find much evidence of it nowadays – this GitHub repo uses "draw.io" pretty much everywhere, the Help menu in the app points to draw.io etc.

So I'd just like to ask for clarification: is the software itself called Draw.io or Diagrams.net?

**Comment by borekb** [comment]
Found some history of this in `README.md` commits:

- https://github.com/jgraph/drawio/commit/f2ca589a180e49a9fe4e59ab392274d220b7fae3 (Apr 2020) changed branding in README from draw.io to diagrams.net

<img width="1085" alt="Screenshot 2023-05-10 at 7 11 57" src="https://github.com/jgraph/drawio/assets/101152/cb2e6dbb-749b-4b61-8659-7938eaa9b20a">

- https://github.com/jgraph/drawio/commit/19bd7cb931a8b43796d9c545c5bf24fb510b9364 (Oct 2021) "reverted" this change:

<img width="1226" alt="Screenshot 2023-05-10 at 7 10 47" src="https://github.com/jgraph/drawio/assets/101152/29f09e63-164c-4de2-a6c4-1a5eeb93e1d4">

**Comment by davidjgraph** [comment]
Either, both, whichever you prefer.
